src/crc32c/__init__.py

```diff
@@ -95,3 +95,19 @@ def copy(self):
         clone = self.__class__()
         clone._crc = self._crc
         return clone
+
+    def consume(self, stream, chunksize):
+        """Consume chunks from a stream, extending our CRC32 checksum.
+
+        Args:
+            stream (readable file-like object): the stream to consume.
```

Comment:
```suggestion
            stream (BinaryIO): the stream to consume.
```

@tseaver There are "generic" descriptions in https://docs.python.org/3/library/typing.html for many such types